### PR TITLE
Add My Invisalign

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: entries/m/my.invisalign.com.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "My Invisalign": {
+    "domain": "my.invisalign.com",
+    "tfa": [
+        "email"
+    ],
+    "categories": [
+        "health"
```
Comment (hkamran80) [inline entries/m/my.invisalign.com.json]:
```suggestion
      "health"
```

File: entries/m/my.invisalign.com.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "My Invisalign": {
+    "domain": "my.invisalign.com",
+    "tfa": [
+        "email"
```
Comment (hkamran80) [inline entries/m/my.invisalign.com.json]:
```suggestion
      "email"
```

Can you provide evidence of 2FA?

**Review by hkamran80 — CHANGES_REQUESTED** [review]

File: entries/m/my.invisalign.com.json (hunk 2 above)
Comment (soodaayush) [reply entries/m/my.invisalign.com.json]: evidence

Evidence of it asking you to check your mail.

**Comment by soodaayush** [comment]
Looks like it is not in the top 200,000 websites on Similarweb, sorry about this.

**Pull request closed without merging**